packages/core/src/components/FacetSearch/FacetSearch.tsx

```diff
@@ -0,0 +1,96 @@
+import { useState } from "react";
+import {
+  HvAccordion,
+  HvCheckBoxGroup,
+  HvCheckBox,
+  HvButton,
+  HvTypography,
+  HvInput,
+} from "@hitachivantara/uikit-react-core";
+import { css } from "@emotion/css";
+import { theme } from "@hitachivantara/uikit-styles";
+
+export const FacetSearch = () => {
```

Comment:
I found the following behaviour with the search:

https://github.com/lumada-design/hv-uikit-react/assets/43220251/3c4af12b-0212-4102-a4ab-fe26a0bffba0

"Coats" is not checked after cleaning the search input